### PR TITLE
Add updates for the 1.15.0 release.

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+
+### Enhancements
+* Added support for accessibility in iOS 12.
+* Updated the visibility checker to support keyboards in iOS 12.
+* Updated Analytics Configurations.
+* Fixed Formatting Issues.
+* Updated invalid api and compatibility docs.
+
+### Compatibility
+* EarlGrey has now been tested for working till Xcode version 10.0 beta 2. Any small unit test breakages with Xcode 9.0's  XCTest Exception are being invesigated.
```
Comment (khandpur) [inline CHANGELOG.md]: double space after 9.0's

invesigated => investigated

also what does this sentence really mean?
"Any small unit test breakages with Xcode 9.0's  XCTest Exception are being invesigated"

why are we seeing issues with 9?

Comment (tirodkar) [reply CHANGELOG.md]: Yeah, XCTestCase Unit Tests. We need to change the exception name for them.

Comment (khandpur) [inline CHANGELOG.md]: Move Xcode 9 stuff into its own bullet point?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
 
 Details changes in each release of EarlGrey. EarlGrey follows [semantic versioning](http://semver.org/).
 
+## [1.15.0](https://github.com/google/EarlGrey/tree/1.15.0) (08/02/2018)
```
Comment (khandpur) [inline CHANGELOG.md]: 08/03/2018

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
+### Enhancements
+* Added support for accessibility in iOS 12.
+* Updated the visibility checker to support keyboards in iOS 12.
+* Updated Analytics Configurations.
+* Fixed Formatting Issues.
+* Updated invalid api and compatibility docs.
+
+### Compatibility
+* EarlGrey has now been tested for working till Xcode version 10.0 beta 2.
+* A few Unit Tests breaks with the XCTest Exception name seen after Xcode 9.3 are being investigated.
```
Comment (khandpur) [inline CHANGELOG.md]: Some of the internal unit tests break on Xcode 9.3+ due to change in exception name thrown by XCTest. Those are still being investigated.

Comment (tirodkar) [reply CHANGELOG.md]: Done.

**Pull request merged**